ludwig/automl/utils.py

```diff
@@ -178,3 +193,20 @@ def set_output_feature_metric(base_config):
         base_config[HYPEROPT]["metric"] = output_metric
         base_config[HYPEROPT]["goal"] = output_goal
     return base_config
+
+
+def check_imbalanced_output(base_config, features_metadata):
```

Comment:
nit: Add type hint that this returns bool.

Reply:
done